packages/js-client/package.json

```diff
@@ -64,7 +57,7 @@
     "@aragon/osx-ethers": "1.3.0-rc0.3",
     "@aragon/sdk-client-common": "1.2.0-rc0",
     "@aragon/sdk-common": "1.5.0",
-    "@aragon/simple-storage-ethers": "1.0.0",
+    "@aragon/simple-storage-ethers": "file:../contracts-ethers",
```

Comment:
Oh, this should not be needed in this repo. Any reason why you added it @josemarinas or was this by accident?